src/DI.php

```diff
@@ -22,6 +22,7 @@
 namespace Friendica;
 
 use Dice\Dice;
+use Friendica\Security\PermissionSet\Depository\PermissionSet;
```

Comment:
Given the large number of single mentions for long namespaced classes, I've been avoiding import like this.
```suggestion
```